utils.js

```diff
@@ -0,0 +1,24 @@
+export function generateId() {
+    return (Date.now().toString(36) + Math.random().toString(36).substr(2, 5))
+}
```

Comment:
Code style comment here again. Just telling you about what I prefer.
No returns needed. seems cleaner. And sometimes you don't have worry about contexts too (I mean, the `this` in arrow functions use the `this` from the parent's context, so yeah, sometimes it means depending on the usage, no bind, call or apply functions to patch up stuff)

```suggestion
export const generateId = () => Date.now().toString(36) + Math.random().toString(36).substr(2, 5);
```

Reply:
Yeah. Arrow functions get confusing to read sometimes and since I wasn't worried about `this` I went with regular functions throughout the code except for the functions in higher-order functions or callbacks.

Is it a good idea to maintain that consistency? Or doesn't matter in your opinion?

Reply:
Again a personal preference, I'm comfortable using arrow functions so I just use them. If you know what you're doing these things wouldn't matter as much. So yeah, I put this is in just to probably let you know about the things that you look into if you chose to write it with arrow functions.

Reply:
Guess the day I find arrow functions easier to read is still far away. :)